[DBS-002] Patch for Zabbix v4.4.0 to remove bracket in operational data
--- zabbix-4.4.5-orig/frontends/php/app/views/monitoring.widget.problems.view.php	2019-10-14 01:13:02.104862978 -0400
+++ zabbix-4.4.5/frontends/php/app/views/monitoring.widget.problems.view.php	2019-10-14 01:23:01.615655216 -0400
@@ -197,7 +197,7 @@
 	];
 
 	if ($show_opdata == OPERATIONAL_DATA_SHOW_WITH_PROBLEM && $opdata) {
-		$problem_link = array_merge($problem_link, [' (', $opdata, ')']);
+		$problem_link = array_merge($problem_link, [' ', $opdata]);
 	}
 
 	$description = (new CCol($problem_link));
--- zabbix-4.4.5-orig/frontends/php/include/blocks.inc.php	2019-10-14 01:17:46.355293284 -0400
+++ zabbix-4.4.5/frontends/php/include/blocks.inc.php	2019-10-14 01:23:46.850563154 -0400
@@ -849,7 +849,7 @@
 			$triggers_hosts[$trigger['triggerid']],
 			getSeverityCell($problem['severity'], null,
 				(($show_opdata == OPERATIONAL_DATA_SHOW_WITH_PROBLEM && $opdata)
-					? [$problem['name'], ' (', $opdata, ')']
+					? [$problem['name'], ' ', $opdata]
 					: $problem['name']
 				)
 			),
--- zabbix-4.4.5-orig/frontends/php/include/classes/screens/CScreenProblem.php	2020-01-09 15:45:41.248520174 +0300
+++ zabbix-4.4.5/frontends/php/include/classes/screens/CScreenProblem.php	2020-01-09 15:46:24.349171059 +0300
@@ -906,10 +906,10 @@
 							->addStyle('width: 42%;'),
 						make_sorting_header(_('Problem'), 'name', $this->data['sort'], $this->data['sortorder'], $link)
 							->addStyle('width: 58%;'),
+						$tags_header,
 						(new CColHeader(_('Duration')))->addStyle('width: 73px;'),
 						(new CColHeader(_('Ack')))->addStyle('width: 36px;'),
-						(new CColHeader(_('Actions')))->addStyle('width: 64px;'),
-						$tags_header
+						(new CColHeader(_('Actions')))->addStyle('width: 64px;')
 					]))
 						->addClass(ZBX_STYLE_COMPACT_VIEW)
 						->addClass(ZBX_STYLE_OVERFLOW_ELLIPSIS);
@@ -931,10 +931,10 @@
 						($show_opdata == OPERATIONAL_DATA_SHOW_SEPARATELY)
 							? _('Operational data')
 							: null,
+						$this->data['filter']['show_tags'] ? _('Tags') : null,
 						_('Duration'),
 						_('Ack'),
-						_('Actions'),
-						$this->data['filter']['show_tags'] ? _('Tags') : null
+						_('Actions')
 					]));
 			}
 
@@ -1085,9 +1085,8 @@
 							->addClass(ZBX_STYLE_WORDWRAP);
 
 						if ($show_opdata == OPERATIONAL_DATA_SHOW_WITH_PROBLEM) {
-							$description[] = ' (';
+							$description[] = ' ';
 							$description[] = $opdata;
-							$description[] = ')';
 						}
 					}
 				}
@@ -1149,14 +1148,14 @@
 						? (new CDiv($description))->addClass('action-container')
 						: $description,
 					($show_opdata == OPERATIONAL_DATA_SHOW_SEPARATELY) ? $opdata : null,
+					$this->data['filter']['show_tags'] ? $tags[$problem['eventid']] : null,
 					($problem['r_eventid'] != 0)
 						? zbx_date2age($problem['clock'], $problem['r_clock'])
 						: zbx_date2age($problem['clock']),
 					$problem_update_link,
 					makeEventActionsIcons($problem['eventid'], $data['actions'], $data['mediatypes'], $data['users'],
 						$this->config
-					),
-					$this->data['filter']['show_tags'] ? $tags[$problem['eventid']] : null
+					)
 				]), ($this->data['filter']['highlight_row'] && $value == TRIGGER_VALUE_TRUE)
 					? getSeverityFlhStyle($problem['severity'])
 					: null
@@ -1186,10 +1185,10 @@
 			_('Host'),
 			_('Problem'),
 			($show_opdata != OPERATIONAL_DATA_SHOW_SEPARATELY) ? _('Operational data') : null,
+			_('Tags'),
 			_('Duration'),
 			_('Ack'),
-			_('Actions'),
-			_('Tags')
+			_('Actions')
 		];
 
 		$tags = makeTags($data['problems'], false);
@@ -1261,15 +1260,15 @@
 				$value_str,
 				implode(', ', $hosts),
 				($show_opdata == OPERATIONAL_DATA_SHOW_WITH_PROBLEM && $trigger['opdata'] !== '')
-					? $problem['name'].' ('.$opdata.')'
+					? $problem['name'].' '.$opdata
 					: $problem['name'],
 				($show_opdata == OPERATIONAL_DATA_SHOW_SEPARATELY) ? $opdata : null,
+				implode(', ', $tags[$problem['eventid']]),
 				($problem['r_eventid'] != 0)
 					? zbx_date2age($problem['clock'], $problem['r_clock'])
 					: zbx_date2age($problem['clock']),
 				($problem['acknowledged'] == EVENT_ACKNOWLEDGED) ? _('Yes') : _('No'),
-				implode(', ', $actions_performed),
-				implode(', ', $tags[$problem['eventid']])
+				implode(', ', $actions_performed)
 			];
 		}
 
